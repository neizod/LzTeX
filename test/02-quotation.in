"she run up to me and asked 'where's your bag?'. i didn't answer and got shot" mark said.
