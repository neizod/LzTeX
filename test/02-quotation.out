``she ran up to me and asked `where's your bag?'. i didn't answer and got shot'' mark said.
